scrc/dataset_creation/law_area_dataset_creator.py

```diff
@@ -0,0 +1,69 @@
+from scrc.dataset_creation.dataset_creator import DatasetCreator
+from scrc.enums.section import Section
+from scrc.utils.log_utils import get_logger
+import numpy as np
+import datasets
+
+from scrc.dataset_creation.report_creator import ReportCreator
+from scrc.utils.main_utils import get_config
+from scrc.enums.split import Split
+
+
+class LawAreaDatasetCreator(DatasetCreator):
+    """
+    Creates a dataset with the facts or considerations as input and the law areas as labels
```

Comment:
When we use the facts as input: we see them as a proxy for the legal complaints. So there the possible application is: The court needs to route incoming complaints to the correct department. 
What is the application when we use the considerations?

You can leave them in, I just don't remember the applicaiton.

Reply:
So you're saying there's an application for that?
I actually didn't have anything specific in mind.

Reply:
Well we need to have an application for the considerations (legal reasoning). Otherwise why are we including them in the dataset?